app/src/main/res/values-es/strings.xml

```diff
@@ -235,64 +275,74 @@
 
   <!-- //////////////// init wallet //////////////// -->
 
-  <string name="initwallet_create">Nueva wallet</string>
-  <string name="initwallet_restore">Restablecer mi wallet</string>
+  <string name="initwallet_create">Crear una nueva cartera</string>
+  <string name="initwallet_restore">Restaurar una cartera</string>
 
   <!-- //////////////// main //////////////////// -->
 
-  <string name="main_alert_electrum_connecting">Conectando al servidor electrum…</string>
+  <string name="main_alert_electrum_connecting">Conectando al servidor Electrum…</string>
   <string name="main_alert_network_lost">No hay conexión. Por favor verifica la configuración de tu red…</string>
-  <string name="main_alert_lightning_connection_lost">No se pudo conectar a la red Lightning…</string>
-  <string name="main_alert_settings_button">Configuración</string>
+  <string name="main_alert_lightning_connection_lost">No se ha podido conectar a la red Lightning…</string>
+  <string name="main_alert_settings_button">Settings</string>
   <string name="main_swapin_incoming">+%1$s entrante</string>
   <string name="main_faq_button">Preguntas frecuentes</string>
+  <string name="main_connectivity_button">Conectando…</string>
+  <string name="main_tor_connected_button">Tor activado</string>
   <string name="main_stores_button">Tiendas</string>
 
   <!-- //////////////// notifications //////////////////// -->
 
-  <string name="inappnotif_set_up_pin">Tu wallet no está protegida.</string>
+  <string name="inappnotif_set_up_pin">Tu cartera no está protegida.</string>
   <string name="inappnotif_set_up_pin_action">Poner un PIN</string>
-  <string name="inappnotif_mnemonics_never_seen">Gracias por usar Phoenix. ¿Ya hiciste una copia de seguridad de tu wallet?</string>
-  <string name="inappnotif_mnemonics_never_seen_action">Crear copia de seguridad de mi wallet</string>
-  <string name="inappnotif_background_worker_cannot_run">Phoenix necesita poder ejecutarse en segundo plano de vez en cuando. Asegúrate de que tu dispositivo no elimine agresivamente esta aplicación para ahorrar batería.</string>
+  <string name="inappnotif_mnemonics_never_seen">Gracias por usar Phoenix. ¿Ya hiciste una copia de seguridad de tu cartera?</string>
+  <string name="inappnotif_mnemonics_never_seen_action">Crear copia de seguridad de mi cartera</string>
+  <string name="inappnotif_background_worker_cannot_run">Phoenix necesita poder ejecutarse en segundo plano de vez en cuando. Asegúrate de que tu dispositivo no fuerce el cierre de esta aplicación para ahorrar batería.</string>
   <string name="inappnotif_upgrade">Hay una actualización disponible para esta aplicación.</string>
-  <string name="inappnotif_upgrade_critical">Una actualización crítica está disponible. Tu wallet puede funcionar mal si no la actualizas.</string>
+  <string name="inappnotif_upgrade_critical">Una actualización crítica está disponible. La cartera puede funcionar mal si se actualiza.</string>
```

Comment:
You missed the word `no`.

Also consider that is the user who updates the wallet; i.e. the wallet is not updated by itself. That's the reason behing `si no la actualizas`.

Reply:
> You missed the word `no`.
> 
> Also consider that is the user who updates the wallet; i.e. the wallet is not updated by itself. That's the reason behind `si no la actualizas`; however that is more like an style and not a technical reason, so go with whatever you consider the best.

Reply:
Well, in this case, I think that is the app wich is updated, so "La **aplicación** puede funcionar mal si **no** se actualiza" is more accurate and follows the previous sentence.